forge-webhook-parser.py

```diff
@@ -5,12 +5,15 @@
 """
 
 import argparse
+import glob
 import json
 import shutil
 import subprocess
+import sys
 from dataclasses import dataclass
 from pathlib import Path
 from typing import List, Optional
+from urllib.request import urlopen
```

Comment:
I typically much rather use https://requests.readthedocs.io/en/latest/ . Even the [urllib.request documentation](https://docs.python.org/3/library/urllib.request.html) says:

> The [Requests package](https://requests.readthedocs.io/en/master/) is recommended for a higher-level HTTP client interface.

But to be honest, I don't have any objective argument why not  to use `urlopen`.